aviary/mission/gasp_based/ode/time_integration_base_classes.py

```diff
@@ -34,7 +34,7 @@ class event_trigger():
     def __init__(
             self,
             state: str,
-            value: tuple[str, float, int],
+            value: tuple,  # [str, float, int],
```

Comment:
Yeah, py3.8 end of life is in October.

Reply:
Whoops, didn't mean for this to make it in, reverted the change now and also upgraded my Python locally :)